CONTRIBUTING.md

```diff
@@ -112,6 +112,30 @@ script will install the task CR locally. For example:
 Currently task tests are not required, so if a task version directory is modified
 in a PR and there are no tests, that directory will be skipped.
 
+##### Testing scenarios where the Task is expected to fail
+
+Sometimes it's desirable to test a negative outcome, for example that given invalid
+parameters for a Task, the resulting TaskRun fails. Currently Tekton does not allow
+to have a task in a pipeline fail and still have the pipeline pass - once a task
```

Comment:
I did not understand the Phrase above. Could you rewrite it?

Reply:
Sorry to hear that, since I really tried to make it understandable 😢  How about something like this?
```
When testing Tasks, most tests will test a positive outcome - that for some input, the task will pass and provide the correct output. But sometimes it's desirable to test that a Task fails when it's expected to fail, for example when invalid data is supplied as input for the Task. But if the Task under test fails in the test Pipeline, the whole Pipeline will fail too. So we need a way to tell the test script that the given test Pipeline is expected to fail. 

You can do this by adding the annotation `test/assert-task-failure`
to the test pipeline object. This annotation will specify which task (`.spec.tasks[*].name`)
in the pipeline is expected to fail. For example:
```

Reply:
I didn't think the paragraph written was too bad, but I think the new version is an improvement on it

Reply:
thanks, @mmalina ! Now it was way easier to understand.

Reply:
Thanks for the feedback. I updated the PR to include the new wording.